### ISSUE TITLE
Not available in the library

### ISSUE THREAD

**Issue by jorgesinval** [lerlerchan/cviJmv#1]
I searched for the package; it does not appear in the library. I also tried to download the repo zip and install it as jmo file. No success.

**Comment by lerlerchan** [comment]
Hi we are still developing the module. The result part is still under development stage. I just push to .jmo to repository. Thank you for noticing this project.

**Comment by jorgesinval** [comment]
Thank you! I want to use it in my classes, some of my students do not enjoy programming in R.

**Comment by jorgesinval** [comment]
By the way, I was able to install it via sideload, but the version...

![image](https://github.com/user-attachments/assets/ddf4e3d3-7894-4b86-925d-b5660d6bbaf4)

I am using Jamovi's latest version (i.e., 2.6.13).

**Comment by lerlerchan** [comment]
Hi currently I am developing on jamovi version 2.5.6 and 2.3.28 (stable version). You may try to install on jamovi version 2.5.6. The issue should resolve.

I will develop the module on jamovi latest version. once the result layout is resolve. result layout is in discussion with jamovi team.

**Comment by jorgesinval** [comment]
Yes, it works with version 2.5.6. This version is not directly available for download on Jamovi's website. For others who want to try your package, they can copy the download link for the latest version and adapt it to the desired version (i.e., 2.5.6).

https://www.jamovi.org/downloads/jamovi-2.5.6.0-win-x64.exe

**Comment by jorgesinval** [comment]
I tried t use it, but it is not working. I first used my data, and then I tried with the sample data, no success:

![image](https://github.com/user-attachments/assets/276be54c-dfd0-48d9-accd-ff21d9aa56f6)

I would be good to have a working example in the repo readme.

All the best.

**Comment by lerlerchan** [comment]
Hi, this is the second open-source module project we are developing. It is nice to receive notice and feedback. We will keep you inform/ update once done.

**Comment by jorgesinval** [comment]
Good! I am more than happy to test it for you.

Sinval
________________________________

**Comment by lerlerchan** [comment]
![image](https://github.com/user-attachments/assets/4a81390c-c2fd-4486-bfc3-840f5331752b)
Hi, Do you need expert in agreement? or proportion relevance data for data analysis?

**Comment by jorgesinval** [comment]
I personally think it's useful to know how many experts agree, but displaying the relative frequencies is more interesting. I hope I understood the question correctly.

Sinval
________________________________
